sigs.yml

```diff
@@ -659,4 +659,24 @@
         github: reyang
     gcLiaison:
       - name: 'Juraci Paixão Kröhling'
-        github: jpkrohling
\ No newline at end of file
+        github: jpkrohling
+    - name: Contributor Experience
+    meeting: TBA
+    notes:
+      type: gDoc
+      value: 1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0
+    chat:
+      - type: slack
+        name: '#otel-contributor-experience'
+        id: C07C5BC6Q6L
+    invites: calendar-contributor-experience
+    sponsors:
+      - name: 'Juraci Paixão Kröhling'
+        github: jpkrohling
+      - name: 'Severin Neumann'
+        github: svrnm
+    gcLiaison:
+      - name: 'Juraci Paixão Kröhling'
+        github: jpkrohling
+      - name: 'Severin Neumann'
+        github: svrnm
```

Comment:
only one of us should be the liaison, cc @jpkrohling

Reply:
I'm fine being the liaison

Reply:
Thanks both! We are going to have to update this doc again once we agree on a meeting time, so I might just leave this PR open for a day or two and wait to re-run all the checks once we have that as well so I'm not doing it twice.